src/uniprotkb/components/statistics/utils.ts

```diff
@@ -38,19 +38,29 @@ export type MergedStatisticsItem = {
 
 export type MergedStatistics = RequireAtLeastOne<
   {
+    uniprotkb?: StatisticsItem;
     reviewed?: StatisticsItem;
     unreviewed?: StatisticsItem;
   },
-  'reviewed' | 'unreviewed'
+  'uniprotkb' | 'reviewed' | 'unreviewed'
 >;
 
 export const mergeToMap = (
+  uniprotkb: StatisticsItem[],
   reviewed: StatisticsItem[],
   unreviewed: StatisticsItem[]
 ) => {
   const accumulator = new Map<string, MergedStatistics>();
+  for (const item of uniprotkb) {
+    accumulator.set(item.name, { uniprotkb: item });
+  }
   for (const item of reviewed) {
-    accumulator.set(item.name, { reviewed: item });
+    const stats = accumulator.get(item.name);
```

Comment:
```
  for (const item of reviewed) {
    const stats = accumulator.get(item.name) || {};
    accumulator.set(item.name, { ...stats, reviewed: item });
  }
  for (const item of unreviewed) {
    const stats = accumulator.get(item.name) || {};
    accumulator.set(item.name, { ...stats, unreviewed: item });
  }
```
I had to think about what is happening when we do `stats.reviewed = item`. I find the above just a little clearer as I know where the `stats.reviewed` will end up. What do you think?